crash_similarity.py

```diff
@@ -122,28 +124,64 @@ def train_model(corpus):
 
     return model
 
+# create distance_matrix using precalculate cosine distance from rwmd
+def create_distance_matrix(model, dictionary, docset, all_distances):
+    distances = np.zeros((len(dictionary), len(dictionary)), dtype=np.double)
+    for j, w in dictionary.items():
+        if w in docset:
+            distances[0:all_distances.shape[1], j] = all_distances[model.wv.vocab[w].index].transpose()
+
+    return distances
+
+
+#Code moodified from Gensim.keyedvector.py
```

Comment:
Write the entire URL (https://github.com/RaRe-Technologies/gensim/blob/4f0e2ae/gensim/models/keyedvectors.py#L339), so we know exactly from which commit this was taken.